lib/commit_summary.bash

```diff
@@ -7,6 +7,7 @@ function commit_summary() {
     local TO="$2"
 
     # print out the commits that have been added
+    git pull --tags
```

Comment:
We need this if we want to use tag names in FROM and TO below. Looks like tags are not always pulled with the code on the agents.